src/behaviors/scrollable.js

```diff
@@ -13,11 +13,15 @@ export default function scrollable(WrappedComponent) {
     }
 
     componentDidMount() {
-      this.node.addEventListener('scroll', this.updateScrollState);
+      if (this.node) {
```

Comment:
So, how much simpler are we talking then? I can get rid of the updateScrollState entirely? Or also the isScrolling state/class?